src/sentry_plugins/slack/plugin.py

```diff
@@ -152,18 +166,18 @@ def notify(self, notification):
 
         # They can be the same if there is no culprit
         # So we set culprit to an empty string instead of duplicating the text
-        if culprit and title != culprit:
+        if self.get_option('include_culprit', project) and culprit and title != culprit:
```

Comment:
Will it be enough to do `self.get_option('include_culprit') == True or self.get_option('include_culprit') == None and ...`? (That seems like it would make the configuration options confusing, with unchecked being either None or False.)

I'm having trouble finding the code that actually calls and handles `get_options`. I had assumed that setting `default` in the configuration would be enough, but I'm not sure when/how that default actually gets applied.

Alternatively - it may be easier to make this opt-in as `exclude_project` and `exclude_culprit` - wdyt?